libs/langchain-google-cloud-sql-pg/src/tests/vectorStore.test.ts

```diff
@@ -0,0 +1,132 @@
+import { test } from "@jest/globals";
+import PostgresVectorStore, { PostgresVectorStoreArgs } from "../vectorStore.js";
+import PostgresEngine, { Column, VectorStoreTableArgs } from "../engine.js";
+import { SyntheticEmbeddings } from "@langchain/core/utils/testing";
```

Comment:
How are you installing this? I am trying to run the test locally but not finding this module